packages/core/src/components/BreadcrumbsBar/__stories__/BreadcrumbsBar.stories.tsx

```diff
@@ -129,21 +123,23 @@ export const WithIcons = {
 
 export const NavigatableBreadcrumbs = {
   render: () => (
-    <div className="monday-storybook-breadcrumbs-bar_inline-wrapper">
+    <Flex gap={12} style={{ padding: "var(--space-8) var(--space-12)", width: 320 }}>
```

Comment:
I don't think it should be identical to the old style, if there's a way to achieve a close look while using standard props it's best